src/Language/PureScript/Ide/Pursuit.hs

```diff
@@ -0,0 +1,110 @@
+{-# LANGUAGE OverloadedStrings    #-}
+{-# LANGUAGE ScopedTypeVariables  #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Language.PureScript.Ide.Pursuit where
+
+import           Prelude                       ()
+import           Prelude.Compat
+
+import qualified Control.Exception             as E
+import           Control.Lens                  hiding (noneOf)
+import           Control.Monad
+import           Data.Aeson
+import           Data.Aeson.Lens
+import           Data.ByteString.Lazy          (ByteString)
+import           Data.Maybe
+import           Data.Text                     (Text)
+import qualified Data.Text                     as T
+import           Language.PureScript.Ide.Types
+import           Network.HTTP.Client           (HttpException (StatusCodeException))
+import           Network.Wreq
+
+import           Text.Parsec
+import           Text.Parsec.Text
+
+instance FromJSON PursuitResponse where
```

Comment:
Sorry, should have read the issue description more thoroughly :)

IMO a large types module is significantly better than an orphan instance.